readme.md

```diff
@@ -135,6 +135,37 @@ const tasks = new Listr([
 It's also possible to return a `stream`. The stream will be converted to an `Observable` and handled as such.
 
 
+#### Skipping tasks
+
+<img src="media/skipped.png" width="255" align="right">
+
+At any point during execution a task can be skipped by invoking the `skip()` function on the task instance passed in to the task function.
+
+```js
+const tasks = new Listr([
+	{
+		title: 'Task 1',
+		task: () => Promise.resolve('Foo')
+	},
+	{
+		title: 'Can be skipped',
+		task: task => {
+			if (someCondition) {
+				task.skip('Reason for skipping');
```

Comment:
I might indeed be more in favour of returning a Symbol. Using a throw might also not work in any case.

```js
task: ctx => {
    return got('https://foo.bar')
        .then(res => {
            if (res.body === 'unicorn') {
                ctx.skip('Reason');
            }

            // handle the other case
        });
        .catch(err => {
            throw new Error('Something went wrong');
        });
}
```

In this case, the promise will swallow the skipped error and the task will fail instead of being skipped. When returning a `symbol` instead, it will still work as expected.

Although while typing this, it won't work either when another `then` was added like this.

```js
task: ctx => {
    return got('https://foo.bar')
        .then(res => {
            if (res.body === 'unicorn') {
                return ctx.skip('Reason');
            }

            return res.body;
        })
        .then(body => {
             // handle the body
        })
        .catch(err => {
            throw new Error('Something went wrong');
        });
}
```